src/app/app.module.ts

```diff
@@ -8,6 +8,11 @@ import { GoogleChartsModule } from 'angular-google-charts'
 
 import { AppComponent } from './app.component'
 import { CadScreenComponent } from './cad-board/cad-board.component'
+// import module
+import { ElModule } from 'element-angular'
+
+// if you use webpack, import style
+import 'element-angular/theme/index.css'
```

Comment:
Could we only include the `css` in the component files?

Because I do not think include `css` into `app.module.ts` is a good idea.

Reply:
ok,I will amend as soon as possible